extensions/libarchive/BinFilesStaticDefinitions.cpp

```diff
@@ -34,19 +34,19 @@ const core::Property BinFiles::MinSize(
 const core::Property BinFiles::MaxSize(
     core::PropertyBuilder::createProperty("Maximum Group Size")
     ->withDescription("The maximum size for the bundle. If not specified, there is no maximum.")
-    ->withType(core::StandardValidators::get().UNSIGNED_LONG_VALIDATOR)->build());
+    ->withType(gsl::make_not_null(&core::StandardValidators::UNSIGNED_LONG_VALIDATOR))->build());
```

Comment:
I think it would make more sense to have the builder functions take a const ref to the validators, instead of a not_null pointer.

Reply:
Yes, that makes the calling site much cleaner.  Fixed in 4648366b31e3f1f1d049e0a9a636dc53791105bb.

I have also changed the constructor of `ConfigurationProperty` in 726fa89d2a54c3918d8936a20765c9e6bb46c454, although we may have to undo that when resolving the conflict with #1543.